### PR TITLE
Test PR commands

### PR DESCRIPTION
**Pull request by seabbs**
This PR tests moving package styling (i.e. stan/C++ checking from @hsbadr and R checks from `styler`) into PR commands run using GitHub actions.

### CODE REVIEW

**Comment by seabbs** [comment]
Testing new PR commands to remove whitespace, style and redocument (/style) and to style C++ code (/cpp).

**Comment by seabbs** [comment]
retest /style

**Comment by hsbadr** [comment]
@seabbs You may also want to restore the automatically-generated `configure` script:
```
# Generated by rstantools.  Do not edit by hand.

#! /bin/sh
"${R_HOME}/bin/Rscript" -e "rstantools::rstan_config()"
```
`rstantools` generally checks if the 1st line includes `Do not edit` before updating the file.

File: .github/workflows/pr-commands.yaml [hunk 1]
```diff
@@ -53,9 +53,9 @@ jobs:
       - uses: r-lib/actions/pr-push@v1
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-  style-stan:
-    if: startsWith(github.event.comment.body, '/style-stan')
-    name: style-stan
+  stan:
+    if: startsWith(github.event.comment.body, '/stan')
+    name: stan
     runs-on: ubuntu-16.04
```
Comment (hsbadr) [inline .github/workflows/pr-commands.yaml:59]: May need to add `sudo apt install clang-format`.

File: .github/workflows/pr-commands.yaml [hunk 2]
```diff
@@ -53,9 +53,9 @@ jobs:
       - uses: r-lib/actions/pr-push@v1
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-  style-stan:
-    if: startsWith(github.event.comment.body, '/style-stan')
-    name: style-stan
+  stan:
```
Comment (hsbadr) [inline .github/workflows/pr-commands.yaml:56]: I'd name it `format` instead of `stan` as it can be used for C/C++ code formatting in any project.

Comment (hsbadr) [reply .github/workflows/pr-commands.yaml:56]: Also, we may use `ubuntu-20.04` instead of `ubuntu-16.04` in all workflows.

Comment (seabbs) [reply .github/workflows/pr-commands.yaml:56]: I updated this to cpp on master (needed to do these changes on master in order to test the GitHub Actions (I am not aware of a better way to test GAs))

File: .github/workflows/pr-commands.yaml (hunk 1 above)
Comment (seabbs) [reply .github/workflows/pr-commands.yaml:59]: good point

File: .github/workflows/pr-commands.yaml (hunk 2 above)
Comment (hsbadr) [reply .github/workflows/pr-commands.yaml:56]: Why did you add it to `pr-commands.yaml` workflow? I'd test with `on push` events for easy and quick triggers. I'm testing a `Lint-Format.yaml` workflow [here](https://github.com/hsbadr/HiClimR).

**Pull request closed without merging**